frontend/app/src/next/hooks/use-runs.tsx

```diff
@@ -181,6 +181,30 @@ export function useRuns({
     },
   });
 
+  const triggerNowMutation = useMutation({
+    mutationKey: ['workflow-run:create', tenant?.metadata.id],
+    mutationFn: async (data: {
+      workflowName: string;
+      input: object;
+      additionalMetadata: object;
```

Comment:
as of now we'll want priority here too I think :P

Reply:
will need to generate and add this throughout